src/site/content/en/blog/display-override/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: Preparing for the display modes of tomorrow
+subhead: PWAs can use the "display_override" property to deal with special display modes.
+authors:
+  - thomassteiner
+date: 2021-02-16
+description: |
+  The display_override property allows developers to define a customized fallback chain of modes
```

Comment:
```suggestion
  The display_override property allows developers to define a customized fallback chain of modes.
```

Reply:
This sentence continues:

The display_override property allows developers to define a customized fallback chain of modes their PWAs should be displayed in.